templates/_helpers.tpl

```diff
@@ -142,22 +142,16 @@ Insert service host environment variables
 {{- define "georchestra.service-envs" -}}
 - name: ANALYTICS_HOST
   value: "{{ include "georchestra.fullname" . }}-analytics-svc"
-- name: ATLAS_HOST
-  value: "{{ include "georchestra.fullname" . }}-atlas-svc"
 - name: CAS_HOST
   value: "{{ include "georchestra.fullname" . }}-cas-svc"
 - name: CONSOLE_HOST
   value: "{{ include "georchestra.fullname" . }}-console-svc"
-- name: EXTRACTORAPP_HOST
```

Comment:
same that upper

Reply:
IMO it makes sense to remove the variables so that the chart reflects the current state of geOrchestra (if the extractorapp has been removed, then there is no reason to keep it here any longer) ; some instances which are using the helm chart can still use a version of the chart which declares these variables ? This is the purpose of versioning the chart actually. But we might have to make it clear which version of the chart is supposed to work with which version of georchestra ?

Reply:
I agree with you, if we want to keep those variable we can put them in extra_environnement  in the specific clusters !